src/quicktype-core/language/CPlusPlus.ts

```diff
@@ -335,6 +335,160 @@ export type TypeContext = {
     inJsonNamespace: boolean;
 };
 
+interface StringType
+{
+    getType(): string;
+    getConstType(): string;
+    getSMatch(): string;
+    getRegex(): string;
+    createStringLiteral(inner: Sourcelike[]): Sourcelike[];
+    wrapToString(inner: Sourcelike[]): Sourcelike[];
+    wrapUtf8FromEncoding(qualifier: Sourcelike[], inner: Sourcelike[]): Sourcelike[];
+    wrapEncodingFromUtf8(qualifier: Sourcelike[], inner: Sourcelike[]): Sourcelike[];
+    wrapEncodingFromUtf8WithType(qualifier: Sourcelike[], t: Type, inner: Sourcelike[]): Sourcelike[];
+    wrapUtf8FromEncodingWithType(qualifier: Sourcelike[], t: Type, inner: Sourcelike[]): Sourcelike[];
+    emitHelperFunctions(): void;
+}
+
+    export function AddQualifier(qualifier: Sourcelike[], qualified: Sourcelike[]): Sourcelike[]
```

Comment:
Functions start with lowercase letters.  Also it seems this doesn't require `export`, and neither do the helper classes below.